packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1402,17 +1416,7 @@ class OffsetLayer extends ContainerLayer {
   ///  * [RenderRepaintBoundary.toImage] for a similar API at the render object level.
   ///  * [dart:ui.Scene.toImage] for more information about the image returned.
   Future<ui.Image> toImage(Rect bounds, { double pixelRatio = 1.0 }) async {
```

Comment:
Link from this doc to `toGpuImage` and briefly point out how they are different?

Reply:
Done. There are some more incidental differences, but we're working on unifying those now.